src/Apps/Collect2/Routes/Collection/Components/CollectionsHubRails/FeaturedCollectionsRails/__tests__/FeaturedCollectionsRails.test.tsx

```diff
@@ -60,7 +54,11 @@ describe("FeaturedCollectionsRails", () => {
   })
 
   it("Renders no arrows when there are less than 5 collections", () => {
-    const component = mount(<FeaturedCollectionsRails {...props} />)
+    const component = mount(
```

Comment:
Now that we're using `MockBoot`, could we add some tests to target different breakpoints, and make sure our logic is correct at each of them?